### PR TITLE
Improvements for Entry Preview (in the context of #7083 and in addition to #7093)

### PR DESCRIPTION
**Pull request by AtrusRiven**
Improvements for integrity and coherence of the entry preview (in the context of #7083; in addition to #7093)

- [ ] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked documentation]

entry-preview_improvements

### CODE REVIEW

File: src/main/java/org/jabref/preferences/JabRefPreferences.java [hunk 1]
```diff
@@ -663,16 +663,17 @@ private JabRefPreferences() {
...
+                        "\\begin{editor & author}<BR><BR>\\format[Authors(LastFirst,FullName,Sep= / ,LastSep= / ),HTMLChars]{\\editor} (\\format[IfPlural(Eds.,Ed.)]{\\editor})\\end{editor & author}__NEWLINE__" +
+                        "\\begin{booktitle}<BR><i>\\format[HTMLChars]{\\booktitle}</i>\\end{booktitle}__NEWLINE__" +
                         "\\begin{chapter} \\format[HTMLChars]{\\chapter}<BR>\\end{chapter}" +
-                        "\\begin{journal}<BR><BR><i>\\format[HTMLChars]{\\journal}</i> \\end{journal} \\begin{volume}, Vol. \\volume\\end{volume}\\begin{series}<BR> \\format[HTMLChars]{\\series}\\end{series}\\begin{number}, No. \\format[HTMLChars]{\\number}\\end{number}__NEWLINE__" +
+                        "\\begin{editor & !author}<BR>\\end{editor & !author}\\begin{!editor}<BR>\\end{!editor}\\begin{journal}<BR><i>\\format[HTMLChars]{\\journal}</i> \\end{journal} \\begin{volume}, Vol. \\volume\\end{volume}\\begin{series}<BR>= \\format[HTMLChars]{\\series}\\end{series}\\begin{number}, No. \\format[HTMLChars]{\\number}\\end{number}__NEWLINE__" +
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/preferences/JabRefPreferences.java:673]: ththere is a`=` sign after the series

Comment (AtrusRiven) [reply src/main/java/org/jabref/preferences/JabRefPreferences.java:673]: Yes, that is intended. I recommend the usage of this sign here to indicate that the following text refers to the series of the publication.

Comment (AtrusRiven) [reply src/main/java/org/jabref/preferences/JabRefPreferences.java:673]: Sorry, forgot to include a screenshot. You can find it in the original PR post.

**Comment by Siedlerchr** [comment]
LGTM so far, I just think that the equal sign confuses users

**Comment by AtrusRiven** [comment]
Thanks for the hint, I removed the "=".

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**